datadog_checks_base/datadog_checks/checks/prometheus/base_check.py

```diff
@@ -93,6 +93,11 @@ def __init__(self, name, init_config, agentConfig, instances=None, default_insta
 
     def check(self, instance):
         endpoint = instance["prometheus_url"]
```

Comment:
Won't this raise a KeyError for old `auto_conf.yaml` configs? We could do something like `endpoint = instance.get('prometheus_url', None)` instead.